torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -988,7 +988,7 @@ RegisterOperators reg(
          noop,
          aliasAnalysisSpecialCase()),
      Operator(
-         "aten::wait(Future(t) self) -> t",
+         "aten::wait(Future(t)! self) -> t",
```

Comment:
This is wrong, `aliasAnalysisSpecialCase()` means `aten::wait` need to be special handled, rather than adding the alias annotation here.